content/docs/addons-test-utils.md

```diff
@@ -104,7 +104,7 @@ afterEach(() => {
 });
 
 it('can render and update a counter', () => {
-  // Test first render and componentDidMount
+  // 測試第一個 render 和 componentDidMount
```

Comment:
We keep original in code comment currently.

Reply:
OK, thanks.
Fix in commit https://github.com/reactjs/zh-hant.reactjs.org/pull/56/commits/ce787f84ead7da09cd235624a12c6c32cdb26390.